entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationRepositoryTest.kt

```diff
@@ -613,6 +614,43 @@ class Neo4jAuthorizationRepositoryTest : WithNeo4jContainer {
         neo4jRepository.deleteEntity(apiaryUri)
     }
 
+    @Test
```

Comment:
you could add a test with one group the user belongs to, and one he does not belong to